README.md

```diff
@@ -166,79 +151,119 @@ A batch file comprises three different line types which exist on a one : many :
 docker-compose build
 ```
 
-### Start
+# How to start the Batch Processor
 
-Use Docker Compose to run service locally.
+The service can be run using the convenience script:
+`./scripts/start`
 
-The service uses [Liquibase](https://www.liquibase.org/) to manage database migrations. To ensure the appropriate migrations have been run the utility script `scripts/start` may be run to execute the migrations, then the application.
+# How to get an output
 
-Alternatively the steps can be run manually:
-* run migrations
-  * `docker-compose -f docker-compose.migrate.yaml run --rm database-up`
-* start
-  * `docker-compose up`
-* stop
-  * `docker-compose down` or CTRL-C
+There are several different possible outcomes depending on the input:
 
-Additional Docker Compose files are provided for scenarios such as linking to other running services.
+1. **Processing a valid payment file**  
+**Input:** A valid payment file.  
+**Output:** The transactions are logged, the payment file is moved from `inbound` to `archive`, and each valid payment request is sent to the Topic `PAYMENT_TOPIC_REQUEST`.  
 
-Link to other services:
-```
-docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
-```
+2. **Processing with the qrong amount of requests**  
+**Input:** A payment file where the amount of payment requests does not match the batch header.  
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.  
+
+3. **Processing a payment file with an invalid batch header**  
+**Input:** A payment file with an invalid batch header (e.g. wrong date format)  
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.  
+
+4. **Processing a payment file with an invalid invoice line total**  
+**Input:** A payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.  
+**Output:** The valid payment requests are logged and error messages are logged for each invalid request, the payment file is moved from `inbound` to `archive`, each valid payment request is sent to Topic `PAYMENT_TOPIC_ADDRESS`, and an invalid event is sent to `EVENT_TOPIC_ADDRESS` for each invalid payment request.  
+
+5. **Processing a payment file with an unexpected sequence number**  
+**Input:** A payment file where the batch header sequence number is different from the expected value.  
+**Output:** If the sequence number is lower than the expected value the payment file is ignored. If the sequence number is higher than the expected value the payment file is quarantined. A message is logged stating that the payment file has been ignored or quarentined accordingly.  
+
+# How to stop the Batch Processor
 
-## Test structure
+This serivice can be stopped in different ways:
+**Bring the service down**  
+`docker-compose down`  
+**Bring the service down and clear its data**  
+`docker-compose down -v`  
```

Comment:
# How to stop the Batch Processor

This service can be stopped in different ways:
- Bring the service down (link to headers below)
- Bring the service down and clear its data

###Bring the service down  

```
docker-compose down
```
  
###Bring the service down and clear its data

```
docker-compose down -v
```